pipeline/api-service/base/config-map.yaml

```diff
@@ -15,6 +15,7 @@ data:
         "AccessKey": "minio",
         "SecretAccessKey": "minio123",
         "BucketName": "mlpipeline",
+        "PipelineFolder": "pipelines",
```

Comment:
nit: in latest code, it was renamed to https://github.com/kubeflow/pipelines/blob/ac66626a2c4c0222158817ac3c920b090af4245b/backend/src/apiserver/client_manager.go#L339, I think the `PipelineFolder` was never released.

Reply:
@Bobgy version 0.2.5 explicitly uses `PipelineFolder` https://github.com/kubeflow/pipelines/blob/0.2.5/backend/src/apiserver/client_manager.go#L331

Reply:
Ohhh, that's a miserable mistake...
I thought their fix went into the same release.

I don't think we should use a release with a wrong env var, that would further confuse next upgrade.
Shall we bump to 0.3.0 instead or directly try to integrate KFP 1.0.0 in https://github.com/kubeflow/manifests/tree/master/pipeline/installs/generic.

Reply:
hmmm, but the cost might be too high for just fixing this issue.
I think this is fine.

When people upgrade, they should have the config overriden automatically.